data/projects/implement_aspects.md

```diff
@@ -1,29 +1,63 @@
+Initial configuration of coala is **hard**, especially for new user. There
+are few reason for this. First is because new user is not always familiar with
+all the functionality provided by various tools inside coala. Beside that, coala
+tries to provide a full set of option for those tools. And also there is a
+possibility that the same type of checking is executed by different tools to
+the same line of code, resulting in duplicated report.
```

Comment:
that would be a bad configuration. The problem is that the some tools have overlapping analysis capabilities so choosing the right one is hard.